packages/affiliates/gas-rebate/VoterGasRebate.js

```diff
@@ -486,7 +486,7 @@ async function getUmaPriceAtTimestamp(timestamp) {
     return ethExchangeRate;
   } catch (err) {
     console.error("Failed to fetch UMA historical price from Coingecko, falling back to default");
-    return 10;
+    return 0.1;
```

Comment:
this is interesting. seems like it should be an env. wont block the pr with this though.

Reply:
ye. this came about because the gas-rebate tests were actually failing on master before but before this PR they were not running in CI. @nicholaspai helped me fix this & this was one of the changes we needed to make.